Samples/UpsertFeature/UpsertFeature/ViewController.swift

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2022 Skyflow
+*/
+
+//
+//  ViewController.swift
+//  App
+//
+//  Created by Tejesh Reddy Allampati on 16/09/21.
+//
+
+
+import UIKit
+import Skyflow
+
+
+
+class ViewController: UIViewController {
```

Comment:
samples/UpsertFeature and samples/Upsert seems to be same. Please remove any one of them

Reply:
removed Upsert folder